.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Description of the Issue:**
+
+**Environment (please complete the following information):**
+ - OS version: <!-- E.g. macOS, Ubuntu 19.04, etc -->
+   Open a terminal and type any of the following commands:
+   ```
+   cat /etc/*release*
```

Comment:
The `cat /etc/*release*` might not be needed now since that'd be included by default in the `nextflow info`.

We can always ask more info, if that's not enough :)

Reply:
Oh yes, you are right. This command will give all the required information.